tests/HistoryServiceSpec.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module HistoryServiceSpec (spec) where
+
+
+import Control.Concurrent.MVar
+import Control.Monad
+import Control.Monad.IO.Class
+import Control.Monad.Trans.RWS.Strict
+import Data.Aeson (Value(Null))
+import Data.ByteString (ByteString)
+import qualified Data.HashMap.Strict as HashMap
+import qualified Data.HashSet as HashSet
+import System.Directory
+import Test.Hspec
+
+import Pact.Server.History.Persistence as DB
+import Pact.Server.History.Service
+import Pact.Server.History.Types
+import Pact.Types.Command
+import Pact.Types.Hash
+import Pact.Types.Server
+import Pact.Types.Term
+
+
+histFile :: FilePath
+histFile = fp ++ "/commands.sqlite"
```

Comment:
Maybe use in memory db, to make tests more robust? https://www.sqlite.org/inmemorydb.html

Reply:
tracking this in #466